### PR TITLE
creating statfullset templates

### PR DESCRIPTION
**Pull request by dhalimi**

### CODE REVIEW

File: openstack/cronus/values.yaml [hunk 1]
```diff
@@ -36,48 +60,18 @@ config:
...
   accountStatusTimeout: 5m
   accountStatusPollDelay: 10s
   verifyEmailDomain: example.com
   verifyEmailSecret: domainkey
   allowedServices:
     email: email-region
 
-# an existing DNS record to forward disco CNAME records
-ingress:
-  record: my-ingress.region.cloud.com
```
Comment (kayrus) [inline openstack/cronus/values.yaml:54]: Can you move this back? The CI is failing because of this.

File: openstack/cronus/values.yaml [hunk 2]
```diff
@@ -36,48 +60,18 @@ config:
...
-
 service:
   type: ClusterIP
-
-# nginx values
-ingress-nginx:
```
Comment (kayrus) [inline openstack/cronus/values.yaml:60]: Can you move this back too?

**Review by kayrus — CHANGES_REQUESTED** [review]
We have a CI job to check the changes, they are failing right now. In addition to save time, before pushing the changes try to test them locally.

File: openstack/cronus/templates/cronus/statefulset.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+          imagePullPolicy: IfNotPresent
+          args:
+            - "-config=/cronus/config.yaml"
+          volumeMounts:
+            - name: cronus-config
+              mountPath: "/cronus/config.yaml"
+              subPath: config.yaml
+          ports:
+            - name: http
+              containerPort: {{ .Values.cronus.http }}
```
[the remark this answers is not in the document]
Comment (dhalimi) [reply openstack/cronus/templates/cronus/statefulset.yaml]: the statefulset file was merged with deployment and the path of the port was fixed in this file

**Review by kayrus — APPROVED** [review]
LGTM, thanks

**Pull request merged**